### DISCUSSION TITLE
Сборка образов

### DISCUSSION THREAD

**Discussion by alglyzin** [mikopbx/Core#50] [General ( Обо всём )]
Добрый вечер. Для того что бы сделать свою модификацию не хватает информации по сборке образов. Возможно ли получить эту информацию?

**Comment by jorikfon** [comment]
Это достаточно сложно, мы думаем как упростить этот процесс и описать.

Если задача стоит по добавлению новых функций - будет в ближайшее время опубликована технология разработки расширений для маркетплейса, бесплатных или платных, на ваш выбор.

Если стоит задача по кастомизации интерфейса, то достаточно в установленной машине смонтировать на запись диск и можно будет менять большую часть кода.
`mount -o remount,rw /offload/`

То что касается загрузки, в надо менять в файле initramfs и переупаковывать его после изменений подбрасывая в образ, тут было немного информации о том как это делать.
https://qa.askozia.ru/6785/запись-вызовов-прерывистый-голос?show=6858#c6858
